test/fixtures/class-decorators-ts/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
 	"compilerOptions": {
+		"moduleResolution": "node",
```

Comment:
`tslib` otherwise can't be found, oddly. This should be the default but maybe rtp2 is doing something odd.